packages/remix/src/index.server.ts

```diff
@@ -6,9 +6,55 @@ import { instrumentServer } from './utils/instrumentServer';
 import { buildMetadata } from './utils/metadata';
 import type { RemixOptions } from './utils/remixOptions';
 
+// We need to explicitly export @sentry/node as they end up under `default` in ESM builds
+// See: https://github.com/getsentry/sentry-javascript/issues/8474
+export {
+  addGlobalEventProcessor,
+  addBreadcrumb,
+  captureException,
```

Comment:
Seems like `captureCheckIn` is missing:

```suggestion
  addBreadcrumb,
  captureCheckIn,
  captureException,
```